index.js

```diff
@@ -61,7 +61,16 @@ say.speak = function(text, voice, speed, callback) {
     pipedData += '(SayText \"' + text + '\")';
   } else if (process.platform === 'win32') {
     pipedData = text;
-    commands = [ 'Add-Type -AssemblyName System.speech; $speak = New-Object System.Speech.Synthesis.SpeechSynthesizer; $speak.Speak([Console]::In.ReadToEnd())' ];
+
+    commands = ['–NoProfile –ExecutionPolicy Restricted;', 
+    'Add-Type -AssemblyName System.speech;', 
+    '$speak = New-Object System.Speech.Synthesis.SpeechSynthesizer;'];
+    try {        
+        var rate = parseInt(speed);
+        commands.push('$speak.rate = ' + (between(rate, -10, 10) ? 0 : rate).toString() + ';');
```

Comment:
The way the `say` API exposes speed is to accept a floating point value where 1.0 is normal speed, 0.5 is half speed, 2.0 is double speed, etc. The platform check and base_speed code is used to normalize this value across platforms, e.g. OS X `say` command uses 175 as normal 1.0 speed, while `festival` uses 100 as normal 1.0 speed.

Could you perform this same normalization for win32? So that 1.0 equates to normal speed, 0.5 half, etc? We do this so that someone using the library could in theory run it on each platform and have the same speech speed without having to do manual checks.